data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,21 @@
+/* exported maxValue */
+
+function maxValue(stack) {
+  let maxValue;
```

Comment:
If you start `maxValue` as `-Infinity` it is helpful and can save you some checks below